packages/table/README.md

```diff
@@ -4,13 +4,6 @@
 
 Use tables to display information in a way that’s easy to scan, so that user can look for patterns and insights. Users will use tables to troubleshoot, manage and monitor their systems.
 
-## Types of Tables
```

Comment:
I think this has been removed by accident?

Reply:
Nope, the "detail table" describes a configuration map, so it felt weird to have it documented in here